core/extension/gdextension_interface.h

```diff
@@ -287,6 +287,29 @@ typedef struct {
 	void *class_userdata; // Per-class user data, later accessible in instance bindings.
 } GDExtensionClassCreationInfo;
 
+typedef void (*GDExtensionClassNotification2)(GDExtensionClassInstancePtr p_instance, int32_t p_what, bool p_reversed);
```

Comment:
Would it make sense to list this immediately below `typedef void (*GDExtensionClassNotification)`, so the differences are obvious?

Reply:
I can change that if needed. Would like to hear an additional opinion, which variant is easier to read. My original thought was to group all new typedefs together.

Reply:
I think https://github.com/godotengine/godot/pull/76577/files is a very clean way of doing it.
Only just realised it is a draft so. @reduz this should be possible, not? @RedworkDE mentions it requires [76446](https://github.com/godotengine/godot/pull/76446) but that is meged.

Reply:
> I can change that if needed. Would like to hear an additional opinion, which variant is easier to read. My original thought was to group all new typedefs together.

Personally, I don't have a strong preference, I think either way is fine. But I'd maybe _slightly_ lean to towards having this `typedef` by the other function pointer `typedefs` :-)

Reply:
I have moved the `typedef` there.